constants.py

```diff
@@ -3,6 +3,7 @@
 BOT_ID = str(os.environ.get('BOT_ID'))
 SLACK_BOT_TOKEN = str(os.environ.get('SLACK_BOT_TOKEN'))
 SLACK_USER_TOKEN = str(os.environ.get('SLACK_USER_TOKEN'))
+CHANNEL_WHITELIST = os.environ.get('CHANNEL_WHITELIST')
```

Comment:
I don't think you should have to wrap any of these in str - they will all be strings unless they are undefined?